src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -46,16 +48,19 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
         this._braintreeSDKCreator.initialize(paymentMethod.clientToken);
         const container = `#${options.containerId}`;
         const messagingContainerId = options.braintreepaypal?.messagingContainerId;
+        const venmoParentContainer = options.containerId;
 
         this._renderButtonsData = {
             paymentMethod,
             paypalOptions,
             container,
             messagingContainerId,
+            venmoParentContainer,
         };
 
         return Promise.all([
-            this._braintreeSDKCreator.getPaypalCheckout({currency: currency?.code}, (paypalCheckoutInstance: PaypalClientInstance) => this.renderButtons(paypalCheckoutInstance)),
+            this._braintreeSDKCreator.getPaypalCheckout({currency: currency?.code}, (paypalCheckoutInstance: PaypalClientInstance) => this._renderButtons(paypalCheckoutInstance)),
+            this._braintreeSDKCreator.getVenmoCheckout((venmoCheckoutInstance: VenmoInstance) => this._renderVenmoButton(venmoCheckoutInstance)),
```

Comment:
Do we need to have a setting-checkbox on the Control Panel which turn on/off Venmo button or this button well be displayed constantly?